web/gds-admin-ui/src/components/LanguageDropdown/index.js

```diff
@@ -1,76 +0,0 @@
-// @flow
-import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
-import { Dropdown } from 'react-bootstrap';
-
-import enFlag from './flags/us.jpg';
-import germanyFlag from './flags/germany.jpg';
-import italyFlag from './flags/italy.jpg';
-import spainFlag from './flags/spain.jpg';
-import russiaFlag from './flags/russia.jpg';
-
-// get the languages
-const Languages = [
-    {
-        name: 'English',
-        flag: enFlag,
-    },
-    {
-        name: 'German',
-        flag: germanyFlag,
-    },
-    {
-        name: 'Italian',
-        flag: italyFlag,
-    },
-    {
-        name: 'Spanish',
-        flag: spainFlag,
-    },
-    {
-        name: 'Russian',
-        flag: russiaFlag,
-    },
-];
-
-const LanguageDropdown = (): React$Element<any> => {
-    const enLang = Languages[0] || {};
-    const [dropdownOpen, setDropdownOpen] = useState(false);
-
-    /*
-     * toggle language-dropdown
-     */
-    const toggleDropdown = ({ dropdownOpen: boolean }) => {
-        setDropdownOpen(!dropdownOpen);
-    };
-
-    return (
-        <Dropdown show={dropdownOpen} onToggle={toggleDropdown}>
-            <Dropdown.Toggle
-                variant="link"
-                id="dropdown-languages"
-                as={Link}
-                to="#"
-                onClick={toggleDropdown}
```

Comment:
Perhaps we could get rid of this for now, or comment it out.

Reply:
I'll put in comment